grammar/abnf-grammar.txt

```diff
@@ -1046,15 +1053,16 @@ member-variable-declarations = *( identifier ":" type ( "," / ";" ) )
 member-function-declaration = function-declaration
 
 ; A circuit declaration defines a circuit type,
-; as consisting of member variables and functions.
+; as consisting of constant members, member variables and functions.
```

Comment:
as consisting of member constants, member variables, and member functions.

OR

as consisting of member constants, variables, and functions.